prov/verbs/src/ep_rdm/verbs_rdm.h

```diff
@@ -307,23 +316,16 @@ struct fi_ibv_rdm_tagged_postponed_entry {
 	struct fi_ibv_rdm_tagged_conn *conn;
 };
 
-enum fi_ibv_rdm_buffer_status {
-	BUF_STATUS_FREE = 0,
-	BUF_STATUS_BUSY
-};
-
+/*
+ * Size of the structure is performance sensitive. TODO: resize to 4 bytes.
+ */
 struct fi_ibv_rdm_buffer_service_data {
 	volatile enum fi_ibv_rdm_buffer_status status;
```

Comment:
This is not part of this patch, but I noticed this as part of the review.  The use of volatile is almost always wrong unless it is a memory address that an IO device can update.